public_dropin_environments/triton_server/start_server.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Copyright 2024 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+echo "Starting Custom Model environment with Triton inference server"
+
+if [ "${ENABLE_CUSTOM_MODEL_RUNTIME_ENV_DUMP}" = 1 ]; then
+    echo "Environment variables:"
+    env
```

Comment:
```suggestion
    env

    echo
    echo "Running NVIDIA init scripts..."
    echo
    /opt/nvidia/nvidia_entrypoint.sh /bin/true
```
What do you think about running these if we are in some sort of debugging mode?

Reply:
It's nice to have. I think, we may want to run it w/o any special env vars. I'll leave it as is for now.

Reply:
And you're using the `/bin/true` here, because of this:
```
# This script can either be a wrapper around arbitrary command lines,
# or it will simply exec bash if no arguments were given
if [[ $# -eq 0 ]]; then
  exec "/bin/bash"
else
  exec "$@"
fi
```

Reply:
Yeah, if you don't have _something_ then it goes into a shell and your container will just hang.